Example/User.swift

```diff
@@ -28,22 +28,22 @@ struct User: Identifiable {
 
 extension User: DictionaryInitializable, DictionaryRepresentable {
     
-    init(dictionary: [String: AnyObject]) throws {
+    init(dictionary: [String: Any]) throws {
         guard let
             id = dictionary["id"] as? String,
-            name = dictionary["name"] as? String,
-            email = dictionary["email"] as? String
-            else { throw JaymeError.ParsingError }
+            let name = dictionary["name"] as? String,
+            let email = dictionary["email"] as? String
+            else { throw JaymeError.parsingError }
         self.id = id
         self.name = name
         self.email = email
     }
     
-    var dictionaryValue: [String: AnyObject] {
```

Comment:
The var is a `[String: Any]` but you are casting the values as `AnyObject` . Shouldn't they be casted as `Any` ?